### PR TITLE
Add data on Health Facilities

### PR DESCRIPTION
**Pull request by Bezzy1999**
As discussed in https://github.com/Code4Nepal/nepalmap_app/issues/149 this pr adds that data.

Let me know what you think, I ended up abbreviating some of the facility types in `health.py` due to some overlap issues on displaying the bar chart.

### CODE REVIEW

**Comment by cliftonmcintosh** [comment]
@Bezzy1999

Thank you very much. This is very nice work and will be a great addition to the data.

Before merging, I would like to do a couple of sanity checks by comparing some data on the original data source with the numbers that you have collected.

File: wazimap_np/health.py [hunk 1]
```diff
@@ -71,6 +95,12 @@ def get_health_profile(geo_code, geo_level, session):
         child_nourishment, _ = child_nourishment_table.get_stat_data(
             geo_level, geo_code, percent=False)
 
+        health_facilities_data, _ = get_stat_data(
+            ['facilitytype'], geo_level, geo_code, session,
+            recode=dict(HEALTH_CENTER_RECODES),
+            key_order=HEALTH_CENTER_RECODES.values(),
```
Comment (cliftonmcintosh) [inline wazimap_np/health.py]: If you use `order_by='-total'` instead of `key_order=HEALTH_CENTER_RECODES.values()`, then the bars will go from largest to smallest every time, like this:

dadeldhura-greatest-to-least

Would you consider updating this pull request or making another pull request to make this change?

File: sql/healthfacilities.sql [hunk 2]
```diff
@@ -0,0 +1,1650 @@
...
+14,district,HEALTH_POST,684
+14,district,PRIVATE_HOSPITAL,0
+14,district,DISTRICT_COLD_ROOM,76
+14,district,ZONAL_HOSPITAL,0
+14,district,DISTRICT_CENTER,0
+14,district,SUB_HEALTH_POST,2736
+14,district,REFUGEE_CAMP,0
+14,district,DISTRICT_AYURVEDIC_HC,0
+14,district,DAHC,0
+14,district,DPHO,76
```
Comment (cliftonmcintosh) [inline sql/healthfacilities.sql]: The numbers for district 14 seem off. There are 684 health posts and 2736 sub health posts listed for district 14. That would be about half of the sub health posts in the country. Can you verify these figures?

File: wazimap_np/health.py (hunk 1 above)
Comment (Bezzy1999) [reply wazimap_np/health.py]: Sure, I'll update this pr.

File: sql/healthfacilities.sql (hunk 2 above)
Comment (Bezzy1999) [reply sql/healthfacilities.sql]: Yes, good spot....I'll check.

Comment (Bezzy1999) [reply sql/healthfacilities.sql]: Yes, there are 2736 unique entries in the shapefile for 'Sub Health Post' and 'Udayapur' however it seems like those entries are just these 36 repeated 76 times:

```
[0, 'Sub Health Post', 'Udayapur', 'Katari', 'E-SAG-14-017'],
[0, 'Sub Health Post', 'Udayapur', 'Risku', 'E-SAG-14-030'],
[0, 'Sub Health Post', 'Udayapur', 'Tawashree', 'E-SAG-14-039'],
[0, 'Sub Health Post', 'Udayapur', 'Okhale', 'E-SAG-14-026'],
[0, 'Sub Health Post', 'Udayapur', 'Limpatar', 'E-SAG-14-023'],
[0, 'Sub Health Post', 'Udayapur', 'Sirise', 'E-SAG-14-035'],
[0, 'Sub Health Post', 'Udayapur', 'ShorungChabise', 'E-SAG-14-033'],
[0, 'Sub Health Post', 'Udayapur', 'Lekhani', 'E-SAG-14-021'],
[0, 'Sub Health Post', 'Udayapur', 'Thanagau', 'E-SAG-14-040'],
[0, 'Sub Health Post', 'Udayapur', 'Basabote', 'E-SAG-14-005'],
[0, 'Sub Health Post', 'Udayapur', 'Tamlida', 'E-SAG-14-037'],
[0, 'Sub Health Post', 'Udayapur', 'Baraha', 'E-SAG-14-003'],
[0, 'Sub Health Post', 'Udayapur', 'Iname', 'E-SAG-14-013'],
[0, 'Sub Health Post', 'Udayapur', 'Barai', 'E-SAG-14-004'],
[0, 'Sub Health Post', 'Udayapur', 'Nametar', 'E-SAG-14-025'],
[0, 'Sub Health Post', 'Udayapur', 'Dumre', 'E-SAG-14-010'],
[0, 'Sub Health Post', 'Udayapur', 'Jante', 'E-SAG-14-015'],
[0, 'Sub Health Post', 'Udayapur', 'Laphagau', 'E-SAG-14-020'],
[0, 'Sub Health Post', 'Udayapur', 'Pokhari', 'E-SAG-14-028'],
[0, 'Sub Health Post', 'Udayapur', 'Valayadanda', 'E-SAG-14-044'],
[0, 'Sub Health Post', 'Udayapur', 'Aaptar', 'E-SAG-14-001'],
[0, 'Sub Health Post', 'Udayapur', 'Khanbu', 'E-SAG-14-019'],
[0, 'Sub Health Post', 'Udayapur', 'Jalpachilaune', 'E-SAG-14-014'],
[0, 'Sub Health Post', 'Udayapur', 'Saune', 'E-SAG-14-032'],
[0, 'Sub Health Post', 'Udayapur', 'Sidhdipur', 'E-SAG-14-034'],
[0, 'Sub Health Post', 'Udayapur', 'Hadiya', 'E-SAG-14-011'],
[0, 'Sub Health Post', 'Udayapur', 'Jogidaha', 'E-SAG-14-016'],
[0, 'Sub Health Post', 'Udayapur', 'Sundarpur', 'E-SAG-14-036'],
[0, 'Sub Health Post', 'Udayapur', 'Bashaha', 'E-SAG-14-006'],
[0, 'Sub Health Post', 'Udayapur', 'Chaudandi', 'E-SAG-14-009'],
[0, 'Sub Health Post', 'Udayapur', 'Katunjebawala', 'E-SAG-14-018'],
[0, 'Sub Health Post', 'Udayapur', 'Tapeswori', 'E-SAG-14-038'],
[0, 'Sub Health Post', 'Udayapur', 'Koshi Tappu Wildlife', 'E-SAG-14-046'],
[0, 'Sub Health Post', 'Udayapur', 'Mainamiani', 'E-SAG-14-024'],
[0, 'Sub Health Post', 'Udayapur', 'Hardeni', 'E-SAG-14-012'],
[0, 'Sub Health Post', 'Udayapur', 'Tribeni', 'E-SAG-14-042'],
[0, 'Sub Health Post', 'Udayapur', 'Katari', 'E-SAG-14-017'],
[0, 'Sub Health Post', 'Udayapur', 'Risku', 'E-SAG-14-030'],
```

I'm not sure whether we should just include these 36?

Comment (Bezzy1999) [reply sql/healthfacilities.sql]: Looking at the map, it does seem like 36 should be the figure; I'll check whether any of the other regions have duplicates like this.

Comment (cliftonmcintosh) [reply sql/healthfacilities.sql]: Removing what appear to be duplicates seems reasonable. 

Several of the other facility types look like they may have repeats in this district, too: hospital, sub center, health post, district cold room, DPHO all have large numbers.

Can you change how the script processes data so it enforces uniqueness?

**Comment by Bezzy1999** [comment]
@cliftonmcintosh I have updated the sql with these changes. I'm still concerned that we may be consolidating locations but the data does look more reasonable now,

I'll add the updated script as a new pr in maptools once you're happy.

Let me know what you think.

**Comment by cliftonmcintosh** [comment]
@Bezzy1999 

Thank you for the update. Looking through it, it does seem that the figures are more likely than some of the ones before. I don't really know how to assess whether we might be off or by how much. I am comfortable with the strategy of enforcing uniqueness, and I am comfortable with the data.
